src/test/e2e/api/admin/feature-archive.e2e.test.ts

```diff
@@ -98,6 +98,18 @@ test('returns three archived toggles', async () => {
         });
 });
 
+test('returns three archived toggles with archivedAt', async () => {
+    expect.assertions(2);
+    return app.request
+        .get('/api/admin/archive/features')
+        .expect('Content-Type', /json/)
+        .expect(200)
+        .expect((res) => {
+            expect(res.body.features.length).toEqual(3);
+            expect(res.body.features[0].archivedAt).toBeTruthy();
```

Comment:
Could check all the features:

```ts
.expect((res) => {
    expect(res.body.features.length).toEqual(3);
    expect(res.body.features.every((f) => f.archived)).toEqual(true);
    expect(res.body.features.every((f) => f.archivedAt)).toEqual(true);
});
```

Reply:
Yeah, I was designing thinking in a mind that a test should usually test single functionality, but either is fine.